php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1080,7 +1080,7 @@ class NDB_BVL_Instrument extends NDB_Page
             $flag = true;
             if (is_array($elvalue)) {
                 foreach ($elvalue AS $val) {
-                    if ($val == "") {
+                    if ($val === "") {
```

Comment:
After a quick check across instruments, it seems like they would have similar returns for other elements